addons/calendar/migrations/11.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,22 @@
+
+---Fields in module 'calendar'---
+calendar     / calendar.contacts        / user_id (many2one)            : now required, default = function
+# Nothing to do, now required but filled by default function
```

Comment:
The default is user specific. Can you set user_id to create_uid where user_id is null in the pre-migration?